extensions/wikia/PageHeader/templates/PageHeaderController_languages.php

```diff
@@ -0,0 +1,24 @@
+<div class="wds-dropdown page-header__languages <?= count( $languages->languageList ) <= 1 ? 'wds-is-disabled' : '' ?>">
+	<div class="wds-dropdown__toggle">
+			<span class="" data-tracking="interwiki-dropdown">
+				<?= $languages->currentLangName ?>
+				<?= DesignSystemHelper::renderSvg(
```

Comment:
According to http://fandomdesignsystem.com/#/components/dropdowns, the `svg` should be outside of the `span` element.